book/team.yaml

```diff
@@ -415,7 +414,29 @@ people:
       - Python
     social:
       - icon: github
-        icon_pack: fab
+      - icon_pack: fab      
```

Comment:
This extra slash seem to have broken the website build. As part of the deployment, there is a script that converts the YAML file to JSON. The added '-' should not be there and made the script fail and hence the check

Related to issue #32